pkg/kubelet/kubelet.go

```diff
@@ -110,6 +111,18 @@ const (
 	housekeepingPeriod = time.Second * 2
 
 	etcHostsPath = "/etc/hosts"
+
+	// Capacity of the channel for recieving pod lifecycle events. This number
+	// is a bit arbitrary and may be adjusted in the future.
+	plegChannelCapacity = 1000
+
+	// TODO(yujuhong): migrate some of the parameters to kubelet flags.
+	// Relisting is used to discover missing container events.
+	// Use a shorter period because generic PLEG relies on relisting for
+	// discovering container events.
+	plegRelistPeriod = time.Second * 3
```

Comment:
Acknowledged!

On Thu, Nov 12, 2015 at 2:10 PM, Yu-Ju Hong <notifications@github.com>
wrote:

> In pkg/kubelet/kubelet.go
> <https://github.com/kubernetes/kubernetes/pull/13571#discussion_r44722406>
> :
>
> > @@ -110,6 +111,18 @@ const (
> >  	housekeepingPeriod = time.Second * 2
> >
> >  	etcHostsPath = "/etc/hosts"
> > +
> > +	// Capacity of the channel for recieving pod lifecycle events. This number
> > +	// is a bit arbitrary and may be adjusted in the future.
> > +	plegChannelCapacity = 1000
> > +
> > +	// TODO(yujuhong): migrate some of the parameters to kubelet flags.
> > +	// Relisting is used to discover missing container events.
> > +	// Use a shorter period because generic PLEG relies on relisting for
> > +	// discovering container events.
> > +	plegRelistPeriod = time.Second * 3
>
> Relisting is still required to populate the cache. The frequency of
> relisting affects how quick kubelet discovers container events (e.g.,
> containers died recently).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13571/files#r44722406>.
>